src/components/AddPlayerForm/AddPlayerForm.spec.js

```diff
@@ -0,0 +1,27 @@
+import { screen, render, fireEvent } from '@testing-library/react'
```

Comment:
I initially used `fireEvent`, too 😁. Jerry suggested to use [`userEvent`](https://github.com/testing-library/user-event) instead, though. `userEvent` includes the individual steps (mouse movement, hover, click, keydown, ...) a user would need to take on the way to the trigger event, whereas `fireEvent`only simulates the actual trigger event.

Reply:
Thank you so much for your nice preview!! i'm impressed how good you do such things...😀
I did take all your advice... (expect the one with the userEvent, because it crushes the test... 😅 I will apply this in the next sprint).